### ISSUE TITLE
Feature request: Add repair data not associated with an event

### ISSUE THREAD

**Issue by JamesPickstone** [TheRestartProject/Fixometer#260]
Request from some users to be able to input repairs made outside specific events.

Useful for:
- Restarters who repair things outside events
- Makerspaces
- Schools

Could we allow Restarters and group admins/hosts to record non-event based data?

**Comment by ngm** [comment]
We have e.g. [Richmond Makerlabs](https://therestartproject.org/groups/richmond-makerlabs/), who periodically create an event in the past to record the ad-hoc, non-event based fixes that they do in the makerspace.  Or e.g. [the Restart Center at Fieldston School](https://therestartproject.org/groups/restart-center-ecfs-2/) who create monthly events and record what they've done within that month.

We don't have anything currently for individual repairs.  At the moment the repair has to come from a group of some kind.

One issue we'd discussed about this in the past was that we would like some level of veracity/data quality indicator to repairs, given that we use it to report impact and fault data.  The repair being recorded by a community repair organisation gives some level of veracity.  Open question: if it is recorded by an individual somewhere in the world, can we report it with the same level of certainty as to its veracity?  As unlikely as it is, we need to think about ensuring we're not open to the repair data being 'gamed' in some way.

With the new developments though, we will have a Restarter profile, and Restarters will likely be associated with a local group, so that gives us a higher level of 'trust' in the data I would say.

Having a Restarter role definitely opens up the possibility of this development.   Needs more discussion though!

e.g:

* it links into the question of as a Restarter, do I want to be linked to individual repairs?  Obviously some people want to, but some might not want to be (e.g. because of privacy/liability reasons.) Perhaps it can be optional.
* from a reporting perspective, would we treat a repair done outside of a community repair event in the same way as one done during an event?

Would be good @JamesPickstone to hear from the users who have requested, their thoughts.  Why do they want it?  (I imagine because it's nice to track your repairs and get your own individual stats?)

**Comment by janetgunter** [comment]
I think individual's own data would need to have a different "quality score" than community data. 

We would also want to encourage individuals to repair with/for other people, to emphasise that we are most interested in "social" repairs. Not just a loner in their workshop fixing things fished out of the rubbish.

I know that John (long-time Camden Restarter) is constantly fixing things for neighbours, family and people in need. (But I can't see him wanting to use this TBH).

**Comment by janetgunter** [comment]
I think this feature could be more seen as a "carbon" (and critical raw material) literacy tool, where the user learns more about the impacts of repairs. If it's a gateway to greater understanding and activism, I'm all for it.

**Comment by ugomatic** [comment]
I agree that "quality score" is an issue, and we should carefully highlight it. I see three interesting types of users who might be interested in this: 
- active Restarters wanting to track the impact (and repairability feedback) of their volunteering beyond Restart Parties
- "online Restarters", active in discussions/micro-tasking/wiki, but not (yet) involved in a group
- brand-new potential volunteers, attracted by reporting a successful (or unsuccessful!) repair

The third type of users is in line with what Fixit Clinic does for people who can't attend an event: http://fixitclinic.blogspot.co.uk/p/item-re.html 

It also resonates with the way other organisations collect feedback from "frustrated users" (ie Test Achats: https://www.test-achats.be/trop-vite-use and BuyMeOnce: https://uk.buymeonce.com/pages/it-broke). Although the quality score would be lower, this could be useful additional data to collect, and could lead to more volunteering, online and offline

**Comment by p-leriche** [comment]
I can see that maintaining data quality is a major issue, both data from people with good intentions simply entering inadequate or inaccurate data, and from others less well intentioned entering "spoiler" data with whatever motivation. Data cleansing is a major headache for any database, especially when the data is entered by diverse groups of people.

We don't have a system of reputation at present on the platform, though we do have badges. A badge or a certain rep could earn the privilege to add fixometer data, but the activities required to earn that privilege would have to be carefully thought through. And maybe a mandatory fixometer training session should also be a requirement.

Personally, I would welcome being able to track my own fixes on the fixometer. Often I get so engrossed in a fix that I wouldn't notice if WW3 broke out, and by the time I'm through with it I frequently struggle to remember what my previous fix was! But the public visibility of fix attribution needs, at least, to be user-controllable. I'd be happy for anyone to see my rep, my skills and my badges but if I thought I'd fixed a device for a punter with an attitude problem and it went up in smoke when the user got home I wouldn't want that user venting his/her spleen at me having identified me through the fixometer.

Regarding Janet's comment about a carbon literacy tool, it would be great to have a publicly accessible "what-if" feature in the fixometer. I imagine most of the required data and technology are in place. It would be nice for people to use it to answer questions like what is the environmental impact of repairing vs upgrading my gadget? Get The Guardian, BBC Click and a few others onto it and it could go viral!

**Comment by Divyastra** [comment]
I raised this question with James on restarters.net platform and he's linked me here and suggested i post it as a future request for wider discussion. I thought i prudent to reopen it here first.

A lot of very good points raised above. The "Quality Score" being the most tricky, Is there any reason that data for reporting aggregates can be treated differently based on quality score? Data must satisfy a quality score to be counted as official. But data that don't meet quality criteria can still be used personally by individual fixers without it skewing the overall data set.

I very much like p-leriche's idea of reputation scores, as difficult as it is to calculate reputation  if that were to be addressed it would go along way to alleviating "reporting quality score issues." would voting for a post or approval from users on the usefulness (more on this in a bit) a repair count towards reputation? Many forums use reputation, trust, and voting to deal with similar questions, though the quality score is an open indication to the reader rather than for qualifying the veracity of contributed data.

As avery basic level of trust, a user who has filled in their profile info in a verified way, who has contributed, to a group or events in a meaningful way, who's name has been tagged with a number of repairs etc could all contribute to reputation and by extension trust in the veracity of entries made by that individual.

Much like some of the examples given above by others, Leicester Hackspace indulges in ad-hoc non event based repairs, in the case of interacting with the public these may happen on a Tuesday Open hack night. Some these repairs can extend over many weeks, so tracking it as an item rather than an event becomes important. Often the Item was first seen at an event run by Leicester Fixers but couldn't be completed there and so the person is invited back to complete the repair on an open hack night, after perhaps purchasing a spare part. Sometimes advise is given online and a repair effected by the end user alone based on that advice. (clearly this might have a low level of quality score but I think it's important to enable such reporting so we can see to what extent this has an impact of community repair).

As items, regardless of whether reported ad-hoc or as a result of being presented at an event. I would very much like to see more information presented about how the repair was done, and how the diagnosis differed from the initial assumptions as presented. Where new techniques are developed for repairs, As to what best practices might be in frequent ly seen repairs. If this becomes part of a searchable library it could possibly give insight in to how others have done repairs which might otherwise have stumped some repairers. ideal the commentary will be supported by photos too. Currently we rely on forums, youtube videos and sites like Fixit for this. However there is a wealth of enabling information to be collected from our own fixers. There is far more time to record such methodology during an independent or ad-hoc repair than might be possible during a busy repair where a fixer is trying to complete a repair quickly because other members of the public are waiting to be seen. Ad-Hoc or independent repairs so recorded with supporting commentary and photographs can easily be assessed for quality score and the repairer awarded reputation for their efforts by readers who found it useful. Much more so than just a simple recording of whether an item was fixed or not.

It is with a cojoining of such widely differing but interconnected feature requests that I would ask that Ad-hoc and individual repairs be reconsidered, the problems of veracity not withstanding as that might be less of an impediment when it is no longer examined in isolation.
ooo-xxx

ooo-xxx
